### PR TITLE
Accelerators - Pipeline text needs to be edited and cut

### PR DESCRIPTION
**Pull request by dbuzhorkaa**
* All external url should open in new tab

### CODE REVIEW

**Comment by bdmogal** [comment]
Can we also change the hero text on the pipeline page to be the same?

**Comment by dbuzhorkaa** [comment]
@bdmogal done. Also added plugins and videos text mentioned in #59

File: content/en/resources/plugins/index.md [hunk 1]
```diff
@@ -3,8 +3,7 @@ title: Plugins
 type: plugins
 hero:
   title: Plugins
-  body: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin commodo iaculis ex, non rhoncus arcu blandit sed. 
-  Fusce eget pharetra nulla, vel fermentum ante. Phasellus gravida non diam sed vehicula."
+  body: "CDAP is highly extensible, and exposes plugins, which allow users to extend its capabilities. On this page, you can see all the plugins available in CDAP. Refer to the community page to learn about writing your own plugin."
```
Comment (bdmogal) [inline content/en/resources/plugins/index.md]: sorry, can you change this slightly - ... Refer to the community page to learn about contributing your own plugin.

Comment (dbuzhorkaa) [reply content/en/resources/plugins/index.md]: done

**Pull request merged**